src/api.js

```diff
@@ -0,0 +1,9 @@
+export default function moveToApi(player) {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Ml0wfxegyFf06umHiVlI/scores/', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify(player),
+  }).then((res) => res.json());
+}
```

Comment:
- [ ] [OPTIONAL] Sorry this wasn't mentioned before however, according to the requirements we are required to make use of ES6 which means you should make use of arrow functions instead of the function keyword. because this wasn't mentioned before this will be an optional change however, I do strongly advise you to adjust it before moving forward.